docs/file-formats/json-project/ontologies.md

```diff
@@ -109,14 +109,16 @@ A resource object needs to have the following fields:
 - `name`
 - `labels`
 - `super`
-- `cardinalities`
+- `cardinalities`*
 
 The following field is optional:
 
 - `comments` 
 
-A detailed description of `resources` can be found [below](#the-property-object-in-detail).
+A detailed description of `resources` can be found [below](#the-resource-object-in-detail).
 
+(*It is technically possible to have a resource without cardinalities,
+but in most cases it doesn't make sense to omit them.)
```

Comment:
isn't there some nice way of doing footnotes in markdown (that would ideally be supported by mkdocs) - that would look nicer, I suppose